gpAux/gpdemo/Makefile

```diff
@@ -9,31 +9,7 @@
 top_builddir = ../..
 -include $(top_builddir)/src/Makefile.global
 
-#
-# VARIABLES
-#
-
-export enable_gpfdist
-export with_openssl
-
-PORT_BASE ?= 7000
-NUM_PRIMARY_MIRROR_PAIRS ?= 3
-BLDWRAP_POSTGRES_CONF_ADDONS := fsync=off $(BLDWRAP_POSTGRES_CONF_ADDONS)
-
-ifeq ($(WITH_MIRRORS), )
-WITH_MIRRORS = true
-endif
-
-ifeq ($(WITH_MIRRORS), true)
-WITH_STANDBY = true
-endif
-
-export DEMO_PORT_BASE=$(PORT_BASE)
-export NUM_PRIMARY_MIRROR_PAIRS
-export WITH_MIRRORS
-export WITH_STANDBY
-export BLDWRAP_POSTGRES_CONF_ADDONS
-export DEFAULT_QD_MAX_CONNECT=150
+SHELL := /usr/bin/bash
```

Comment:
Why add this?

Reply:
because `source` is a bash thing. only in this way can we provide defaults for `gpAux/gpdemo/Makefile`.